src/resources/elements/primeDesignSystem/prange-slider/prange-slider.ts

```diff
@@ -10,33 +10,39 @@ export class PRangeSlider {
   @bindable leftLabel?: string;
   @bindable rightLabel?: string;
   @bindable maxValue = 100;
+  @bindable({defaultBindingMode: bindingMode.twoWay}) left: number;
+  @bindable({defaultBindingMode: bindingMode.twoWay}) right: number;
   @bindable.ref rangeInput: HTMLInputElement;
 
   constructor(public element: Element) {
   }
 
-  @computedFrom("value")
-  get percentage() {
-    this.updateCssVariables();
-    return this.value ?? 0;
-  }
-
   @computedFrom("value", "maxValue")
   get leftValue() {
-    return this.percentageToAbsoluteValue(this.value);
+    return this.left = this.percentageToAbsoluteValue(this.value);
```

Comment:
Is there another way to update `this.left`?
I'm feeling a bit uncomfortable with this side effect in this getter here.

Reply:
Any recommendation on where to put it? I couldn't find a place that shows the input value in the component but also changes the `this.left` property